<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CardioWave - viewer</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>341</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>./data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Path:</string>
    </property>
    <property name="buddy">
     <cstring>pushButton</cstring>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>10</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>320</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_fix">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>510</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Fit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>520</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Re-Calculate</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="gview">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>370</y>
      <width>400</width>
      <height>300</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="table2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>550</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>14</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>75</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string>Max</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>Valley</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>Peaks</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>Lambda</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>Freq</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>L-std</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>Up</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>FullDown</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>MaxPeak</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>Down</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>RMS</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>P-Uni</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>Amplitude</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>A-std</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>Tail</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>shoulder</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>sh.t.ratio</string>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string>PW10</string>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="9" column="2">
     <property name="text">
      <string>stdPW10</string>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string>PW25</string>
     </property>
    </item>
    <item row="10" column="2">
     <property name="text">
      <string>stdPW25</string>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string>PW50</string>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="11" column="2">
     <property name="text">
      <string>stdPW50</string>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string>PW80</string>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="12" column="2">
     <property name="text">
      <string>stdPW80</string>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string>PW90</string>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="13" column="2">
     <property name="text">
      <string>stdPW90</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>680</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>651</width>
      <height>371</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Field:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_add_filter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Filters</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_delete_filter">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter and select</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_2</cstring>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_select">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>600</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="plainText">
     <string>state=Chronic
compound=Doxorubicin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quick Filters:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_select_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>420</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>75</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>18</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>390</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>concentration</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>well</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>plate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>state</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select waveform</string>
    </property>
    <property name="buddy">
     <cstring>comboBox_3</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_add_filter_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>690</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Filters</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Show 10s</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_quick_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quick Pick</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Filters:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>26</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionThis_waveform"/>
    </widget>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuLab">
     <property name="title">
      <string>Lab</string>
     </property>
     <addaction name="action_fft"/>
     <addaction name="action_welch"/>
    </widget>
    <addaction name="menuLab"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionBug_Report"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionThis_waveform">
   <property name="text">
    <string>This waveform</string>
   </property>
  </action>
  <action name="actionBug_Report">
   <property name="text">
    <string>Bug Report</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About CardioWave</string>
   </property>
  </action>
  <action name="action_fft">
   <property name="text">
    <string>FFT</string>
   </property>
  </action>
  <action name="action_welch">
   <property name="text">
    <string>Welch</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>graphicsView</tabstop>
  <tabstop>bt_fix</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>gview</tabstop>
  <tabstop>table2</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>graphicsView_2</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>bt_add_filter</tabstop>
  <tabstop>bt_delete_filter</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>listWidget_2</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>bt_select</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>bt_select_2</tabstop>
  <tabstop>tableWidget_2</tabstop>
  <tabstop>comboBox_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
